BE/OwnershipAndControl/Executives.rdf

```diff
@@ -127,25 +127,81 @@
 		<fibo-fnd-utl-av:synonym>articles of association</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-exec;Auditor">
-		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;ResponsibleParty"/>
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;AuthorizedParty"/>
 		<rdfs:label>auditor</rdfs:label>
-		<skos:definition>an individual qualified (at the state level) to conduct audits. An auditor may be an internal auditor (an individual whose primary job function is to audit his or her own company) or an external auditor (an individual from outside the company, who typically is employed by an auditing firm who handles many different clients).</skos:definition>
-		<skos:editorialNote>They have power independently of the board. Power to assess the quality of the financial reports. Also has obligations to the statutory authorities in that regard. The auditor has obligations to the shareholders which are not directly to the Board. Their specific role is to supervise the board to ensure they are acting on behalf of the shareholders. To report to the shareholders on the activities of the Board. Therefore, Auditor&apos;s report is a required part of the financial reports. They are responsible for providing a true and fair view of the financial positions of the company. Election - Put up by the board. Appointed by the Board on behalf of the shareholders. Can be voted out by the shareholders at an AGM or EGM but the board can also put up a competitive process. Normally board elects them but the shareholders can override that selection.</skos:editorialNote>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.investorwords.com/326/auditor.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>party qualified and authorized to review and verify the accuracy of financial records and ensure that companies comply with tax laws</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>An auditor may be an internal auditor - an individual whose primary job function is to audit his or her own company, or an external auditor - an individual from outside the company, who typically is employed by an auditing firm who handles many different clients.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-be-oac-exec;Authorization">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;Situation"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-be-oac-exec;hasAuthorizedParty"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-oac-exec;AuthorizedParty"/>
```

Comment:
Something that we have found in doing some mapping for Wells Fargo is the advantage of using the same (or similar) pattern for lots of relations.   While it is possible to mix 3-class and 5-class patterns, it is fussy.  For example, WF represents all the relations it knows (ownership, beneficial ownership, signatory power, control, and a dozen others) as being between one customer and another, where the customer is clearly a relative thing (it has its own number; they don't use the LEI or the DUNS for this).  So we manage a network of relative things (that's what they are already doing).  If we represent some of the relations as 3-class, we don't have the relative things to link it, so we would have to combine independent things with the relative things.  This is probably more accurate (after all, this entity owns that entity, regardless of whether either or both are our customers), but conceptually difficult - when am I talking about the customer and when am I talking about the entity?  If we use the same 5-class pattern for all of them, we at least have a place for the corresponding things to be.   So, while it isn't conceptually necessary, it makes it easier to compare and combine data about different kinds of relationships.

Reply:
At WF (and at at least one other bank that is a FIBO user), integrating information about the relationships between customers is very important.  But what their internal models and data sources do is relate roles to one another, rather than independent people to one another.  Authorization, is critical for a number of cases, including but not limited to role-based access to various resources (not limited to the security issues around that), access to restricted information related to a customer, such as information subject to privacy concerns, etc.  Having a consistent pattern for these things is really important, and so leveraging the lattice pattern allows us to ensure that consistency.  Control relations in FIBO are still a mess.  Ownership is in much better shape now, though we probably need to do a bit more there.  But I need time to really test that, and so I don't want to attempt to get something in for this upcoming release (i.e., Sunday).

Reply:
I don't disagree with what you both wrote but:
a) what we agreed as a policy was to use the pattern related to the specific modeling scenario rather than based on consistency among many
b) Customer is not a relative thing for this authorization situation but for the customer relationship. You do not have a relative thing purely for Authorizer but reuse the different Customer relative thing. Fine but not something we've documented any sort of pattern for. This is NOT the 5 object (or even the 3 object) pattern, but something different. I tried to catalog different potential patterns (with diagrams and everything) for the FLT but was roadblocked be people "needing more time to review"
c) IMO we cannot accept as justification undocumented use cases/scenarios which are confidential/secret! Either document them properly or don't use them as justification. How can anyone challenge whether what's proposed is the best model for the use case if it's not transparent? You might as well just say: "trust me, I know what is best because I'm omniscient".